models/field_definition_blackbox_test.go

```diff
@@ -1,18 +1,23 @@
-package models
+package models_test
 
 import (
 	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
+
+	. "github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
 )
 
 func TestListFieldDefMarshalling(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
```

Comment:
I'm really unclear on what the concept behind "require" should be. "Database" is a resource we need to execute certain tests. But "UnitTest" is not a resource that can be present or absent. It's more of a label. Is there ever a reason do skip this test? I don't think so.